charts/valheim/values.yaml

```diff
@@ -0,0 +1,64 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/charts/tree/master/charts/common/values.yaml
+#
+
+image:
+  repository: lloesche/valheim-server
+  pullPolicy: IfNotPresent
+  tag: latest
+
+strategy:
+  type: Recreate
+
+env: {}
```

Comment:
I'd include a link to the available environment variables, for example see here

https://github.com/k8s-at-home/charts/blob/master/charts/unifi-poller/values.yaml#L16

Reply:
Added with latest commit.